pkg/crc/config/secret_config.go

```diff
@@ -0,0 +1,69 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/spf13/cast"
+	"github.com/zalando/go-keyring"
+)
+
+const CRCSecretServiceName = "crc"
```

Comment:
I would drop `CRC` from the name, and this can be private to this module, `secretServiceName`